src/specfun.jl

```diff
@@ -26,3 +26,8 @@ gamma(x::Float128) =
     Float128(@ccall(libquadmath.tgammaq(x::Cfloat128)::Cfloat128))
 lgamma(x::Float128) =
     Float128(@ccall(libquadmath.lgammaq(x::Cfloat128)::Cfloat128))
+
+function logabsgamma(x::Float128)
+    sign = x >= 0 ? 1 : 2*mod(ceil(Int64,x),2)-1
```

Comment:
avoid throwing for Inf/NaN
```suggestion
    if isfinite(x)
      sign = x >= 0 ? 1 : 2*mod(ceil(Int64,x),2)-1
    else
      sign = 1
    end
```